File: dev-docs/snippets/documents/has-local-copy.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Check whether a document has a local copy stored on this device.
export function hasLocalCopy(client: JsBaoClient, documentId: string) {
  // #region example
  const local = client.documents.hasLocalCopy(documentId);
  // #endregion example
  return local;
}

File: dev-docs/snippets/documents/has-local-copy.swift
import JsBaoClient

// Check whether a document has a local copy stored on this device.
func hasLocalCopy(client: JsBaoClient, documentId: String) {
  // #region example
  let local = client.documents.hasLocalCopy(documentId: documentId)
  // #endregion example
  _ = local
}
